src/app/main.js

```diff
@@ -13,6 +14,12 @@ Vue.use( PrismicVue, {
 	htmlSerializer
 });
 
+// Init Google Analytics
+Vue.use( VueAnalytics, {
+	id: 'UA-129305382-1',
```

Comment:
Let's change this to something else so that we don't mess with the live account data. I changed it to `UA-XXXXXXXXX-X` and it didn't throw an error so maybe we can change it to this and then switch it out when we're ready to go live. We could also setup a test account.

Reply:
Yep, good point. 6e6728b switches the account (and we'll need some reminder to switch it back again)